chapters/equations.tex

```diff
@@ -149,18 +154,20 @@ \subsubsection{Implicit Iteration Ranges of For-Equations}\label{implicit-iterat
 
 \subsection{Connect-Equations}\label{connect-equations}
 
-A connect-equation has the following syntax:
+A \lstinline!connect!-equation has the following syntax:
 \begin{lstlisting}[language=grammar]
 connect "(" component-reference "," component-reference ")" ";"
 \end{lstlisting}
 
-These can be placed inside for-equations and if-equations; provided the indices of the for-loop and conditions of the if-clause are parameter expressions that do not depend on \lstinline!cardinality!, \lstinline!rooted!, \lstinline!Connections.rooted!, or \lstinline!Connections.isRoot!.  The for-equations/if-equations are expanded.  Connect-equations are described in detail in \cref{connect-equations-and-connectors}.
+These can be placed inside \lstinline!for!-equations and \lstinline!if!-equations; provided the indices of the \lstinline!for!-loop and conditions of the \lstinline!if!-equation are parameter expressions that do not depend on \lstinline!cardinality!, \lstinline!rooted!, \lstinline!Connections.rooted!, or \lstinline!Connections.isRoot!.
+The \lstinline!for!-equations/\lstinline!if!-equations are expanded.
+\lstinline!connect!-equations are described in detail in \cref{connect-equations-and-connectors}.
 
 The same restrictions apply to \lstinline!Connections.branch!, \lstinline!Connections.root!, and \lstinline!Connections.potentialRoot!; which after expansion are handled according to \cref{equation-operators-for-overconstrained-connection-based-equation-systems1}.
 
 \subsection{If-Equations}\label{if-equations}
 
-If-equations\index{if@\robustinline{if}!equation}\index{then@\robustinline{then}!if-equation}\index{else@\robustinline{else}!if-equation}\index{elseif@\robustinline{elseif}!if-equation} have the following syntax:
+\lstinline!if!-equations\index{if@\robustinline{if}!equation}\index{then@\robustinline{then}!if-equation@\robustinline{if}-equation}\index{else@\robustinline{else}!if-equation@\robustinline{if}-equation}\index{elseif@\robustinline{elseif}!if-equation@\robustinline{if}-equation} have the following syntax:
```

Comment:
```suggestion
The \lstinline!if!-equations\index{if@\robustinline{if}!equation}\index{then@\robustinline{then}!if-equation@\robustinline{if}-equation}\index{else@\robustinline{else}!if-equation@\robustinline{if}-equation}\index{elseif@\robustinline{elseif}!if-equation@\robustinline{if}-equation} have the following syntax:
```

Reply:
I must say that I think the suggestion might make it look better, but the original way of starting the sentence with "if" makes it read better.  Anyway, I'm applying the change just so that we can move forward on this.